cosmos.v1.proto

```diff
@@ -0,0 +1,482 @@
+syntax = "proto3";
+
+package cosmos.v1;
+
+option go_package = "github.com/figment-networks/tendermint-protobuf-def/cosmos";
+
+import "codec.proto";
+import "gogo.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
I know in codec.proto we didn't need this, may not be needed here either (it'll be removed in the TM codecs at a later date)

Reply:
I just checked it and this import has to be here, otherwise I got error:
`proto:463:8: "google.protobuf.MessageOptions" seems to be defined in "google/protobuf/descriptor.proto", which is not imported by "cosmos.v1.proto".  To use it here, please add the necessary import.`